react-native-purchases-ui/android/build.gradle

```diff
@@ -0,0 +1,97 @@
+buildscript {
+  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
+  def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["PurchasesUi_kotlinVersion"]
+
+  repositories {
+    google()
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath "com.android.tools.build:gradle:7.2.1"
```

Comment:
Hmm I guess this comes out directly from the tool? It's a relatively old version of the gradle plugin... But I guess we can keep it for now.

Reply:
Yes... I was going to update it but then noticed the main library is using `4.0.1` so I was like this is fine compared to 4.0.1 lol